loafer/ext/aws/consumer.py

```diff
@@ -20,15 +20,16 @@ def __init__(self, source, endpoint_url=None, use_ssl=True, options=None, loop=N
         self._client = None
```

Comment:
I didn't understand what you mean with that, could you please explain it better?